dashboards/rabbitmq/README.md

```diff
@@ -8,4 +8,13 @@
 |RabbitMQ Overview|
 |:------------------|
 |Filename: [rabbitmq-gce-overview.json](rabbitmq-gce-overview.json)|
-|This dashboard has charts displaying `Consumers`, `Current Messages`, `Messages Published`, `Messages Delivered`, `Acknowledged Messages`, `Dropped Messages`, `CPU % Top 5 VMs`, `Memory % Top 5 VMs` and `Hosts by Region`. There is also a card with links to docs and rabbitmq log in Cloud Logging.|
\ No newline at end of file
+|This dashboard has charts displaying `Consumers`, `Current Messages`, `Messages Published`, `Messages Delivered`, `Acknowledged Messages`, `Dropped Messages`, `CPU % Top 5 VMs`, `Memory % Top 5 VMs` and `Hosts by Region`. There is also a card with links to docs and rabbitmq log in Cloud Logging.|
+
+#### Notes
```

Comment:
The Notes header should only appear once at the top of the file. I would suggest adding the notes content here to the dashboard description directly.